sormas-app/app/src/main/res/layout/fragment_person_edit_layout.xml

```diff
@@ -27,6 +27,7 @@
     <data>
         <import type="de.symeda.sormas.api.person.PresentCondition" />
         <import type="de.symeda.sormas.api.person.OccupationType" />
+        <import type="de.symeda.sormas.api.person.ArmedForcesRelationType" />
```

Comment:
It's not needed to import the enum here, it is not used.

Reply:
Alright, removed.